app/subscriber/src/components/content-list/ViewOptions.tsx

```diff
@@ -0,0 +1,81 @@
+import { TooltipMenu } from 'components/tooltip-menu';
+import React from 'react';
+import { FaGear } from 'react-icons/fa6';
+import { Checkbox, Col, Radio, Row } from 'tno-core';
+
+import { IGroupByState, IToggleStates } from './interfaces';
+import * as styled from './styled';
+
+export interface IViewOptionsProps {
+  viewStates: IToggleStates;
```

Comment:
The passing of state props this way is discouraged.  Use context if these properties are required.

Reply:
Roger to the above 2 comments, will update